topography/gaussian_topog.F90

```diff
@@ -39,12 +39,22 @@ module gaussian_topog_mod
 use constants_mod, only: pi
 
 use mpp_mod,       only: input_nml_file
+use platform_mod,  only: r4_kind, r8_kind
 
 implicit none
 private
 
 public :: gaussian_topog_init, get_gaussian_topog
 
+interface gaussian_topog_init
+   module procedure gaussian_topog_init_r4
+   module procedure gaussian_topog_init_r8
+end interface gaussian_topog_init
+
+interface get_gaussian_topog
+    module procedure get_gaussian_topog_r4, get_gaussian_topog_r8
+end interface get_gaussian_topog
+
 !-----------------------------------------------------------------------
```

Comment:
Comments below here look to be html markup and not doxygen.

Reply:
fixed with 98d5ee37